docs/restic.md

```diff
@@ -40,6 +40,21 @@ cross-volume-type data migrations. Stay tuned as this evolves!
 
 1. Run one of the following for your platform to create the daemonset:
 
+   Please note: In RancherOS , the path is not `/var/lib/kubelet/pods` , rather it is `/opt/rke/var/lib/kubelet/pods`
+   thereby requires modifying the restic-daemonset.yaml before applying.
+
+   ```
+   hostPath:
+   path: /var/lib/kubelet/pods
+   ```
+
+   to 
+
+   ```
+   hostPath:
+   path: /opt/rke/var/lib/kubelet/pods
+   ```
+
```

Comment:
Just wondering if it would be a good idea to add a directory for rancher os, just like aws, azure etc.

Reply:
> just like aws, azure etc.

I'm not sure which directories you're referring to for this. The examples directory?

Reply:
yes @nrb, the examples directory that can includes restic ds YAML with the modified path for rancher os. However a generic instruction with Rancher as example also is fine.

Reply:
@kmova I think that would be fine, but I think this can be merged as is, too.